### PR TITLE
[CNVS Upgrade] Fix task log viewer

### PR DESCRIPTION
**Pull request by jfurrow**
This PR applies the correct classes for the log viewer.

### CODE REVIEW

File: src/js/components/FilterInputText.js [hunk 1]
```diff
@@ -92,15 +92,15 @@ class FilterInputText extends React.Component {
     });
 
     return (
-      <span className="form-control-group-add-on form-control-group-add-on-append">
+      <span className="form-control-group-add-on">
```
Comment (mlunoe) [inline src/js/components/FilterInputText.js]: I still see the `form-control-group-add-on-append` and `form-control-group-add-on-prepend` in the Form styles for reactjs-components.
Should these styles live here or there? If here, then we should probably remove them from reactjs-components or the other way around.

Comment (jfurrow) [inline src/js/components/FilterInputText.js]: Good call, I didn't realize that. I just took a quick glance at what's in `reactjs-components` and I believe that it should be removed from `reactjs-components`. The classname I'm using here comes from CNVS.

Comment (mlunoe) [inline src/js/components/FilterInputText.js]: I still see them those classes in `FormsTabContent`, though.

Comment (jfurrow) [inline src/js/components/FilterInputText.js]: The `FormsTabContent` component is part of @ashenden style guide; I suspect it's just leftover from the previous Canvas documentation. The `form-control-group-add-on*` family of classes should be styled via CNVS, not via some custom styling we added to reactjs-components, in my opinion.

In any case, the `form-control-group-add-on-append` class in reactjs-components has no properties that are being applied here, so it should be removed even if we decide to keep the custom styles in reactjs-components.

Comment (mlunoe) [inline src/js/components/FilterInputText.js]: Ok, thanks for checking!
I missed that that was part of CNVS.

**Pull request merged**